packages/tests-unit/tests/core/routing/cacheInterceptor.test.ts

```diff
@@ -0,0 +1,238 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+import { cacheInterceptor } from "@opennextjs/aws/core/routing/cacheInterceptor.js";
+import { convertFromQueryString } from "@opennextjs/aws/core/routing/util.js";
+import { Queue } from "@opennextjs/aws/queue/types.js";
+import { InternalEvent } from "@opennextjs/aws/types/open-next.js";
+import { fromReadableStream } from "@opennextjs/aws/utils/stream.js";
+import { vi } from "vitest";
+
+vi.mock("@opennextjs/aws/adapters/config/index.js", () => ({
+  NextConfig: {},
+  PrerenderManifest: {
+    routes: {
+      "/albums": {
+        initialRevalidateSeconds: false,
+        srcRoute: "/albums",
+        dataRoute: "/albums.rsc",
+      },
+      "/revalidate": {
+        initialRevalidateSeconds: 60,
+        srcRoute: "/revalidate",
+        dataRoute: "/revalidate.rsc",
```

Comment:
You set it as if it were an app router page here, but then you use it as a page router page in test below.
It doesn't matter for the test but for consistency it should be correct here